scalding-core/src/main/scala/com/twitter/scalding/typed/TypedPipe.scala

```diff
@@ -928,7 +930,7 @@ class TypedPipeFactory[T] private (@transient val next: NoStackAndThen[(FlowDef,
 /**
  * This is an instance of a TypedPipe that wraps a cascading Pipe
  */
-class TypedPipeInst[T] private[scalding] (@transient inpipe: Pipe,
+class TypedPipeInst[T] private[scalding] (@transient val inpipe: Pipe,
```

Comment:
let's not add the val, but a `private[scalding] def getPipe: Pipe` to be 100% clear this is not a public method.